src/contracts/core/StrategyManager.sol

```diff
@@ -269,33 +290,33 @@ contract StrategyManager is
     // INTERNAL FUNCTIONS
 
     /**
-     * @notice This function adds `shares` for a given `strategy` to the `staker` and runs through the necessary update logic.
-     * @param staker The address to add shares to
+     * @notice This function adds `nonNormalizedShares` for a given `strategy` to the `staker` and runs through the necessary update logic.
+     * @param staker The address to add nonNormalizedShares to
      * @param token The token that is being deposited (used for indexing)
-     * @param strategy The Strategy in which the `staker` is receiving shares
-     * @param shares The amount of shares to grant to the `staker`
-     * @dev In particular, this function calls `delegation.increaseDelegatedShares(staker, strategy, shares)` to ensure that all
-     * delegated shares are tracked, increases the stored share amount in `stakerStrategyShares[staker][strategy]`, and adds `strategy`
+     * @param strategy The Strategy in which the `staker` is receiving nonNormalizedShares
+     * @param nonNormalizedShares The amount of nonNormalizedShares to grant to the `staker`
+     * @dev In particular, this function calls `delegation.increaseDelegatedShares(staker, strategy, nonNormalizedShares)` to ensure that all
+     * delegated nonNormalizedShares are tracked, increases the stored share amount in `nonNormalizedStakerStrategyShares[staker][strategy]`, and adds `strategy`
      * to the `staker`'s list of strategies, if it is not in the list already.
      */
-    function _addShares(address staker, IERC20 token, IStrategy strategy, uint256 shares) internal {
+    function _addShares(address staker, IERC20 token, IStrategy strategy, uint256 nonNormalizedShares) internal {
         // sanity checks on inputs
         require(staker != address(0), "StrategyManager._addShares: staker cannot be zero address");
-        require(shares != 0, "StrategyManager._addShares: shares should not be zero!");
+        require(nonNormalizedShares != 0, "StrategyManager._addShares: nonNormalizedShares should not be zero!");
 
-        // if they dont have existing shares of this strategy, add it to their strats
-        if (stakerStrategyShares[staker][strategy] == 0) {
+        // if they dont have existing nonNormalizedShares of this strategy, add it to their strats
+        if (nonNormalizedStakerStrategyShares[staker][strategy] == 0) {
```

Comment:
There is so much diff here that doesn't contribute to the implementation in any way.

Let's keep the original variable names here.